butterfly-facade/src/main/java/com/paypal/butterfly/facade/Configuration.java

```diff
@@ -14,6 +14,7 @@
     // See the setters for information about each property
     private File outputFolder = null;
     private boolean zipOutput = false;
+    private boolean modifyInPlace = false;
```

Comment:
Please, rename this to `modifyOriginalFolder`, then rename setter and getter accordingly.